src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -237,20 +237,57 @@ public StudentParticipation startExercise(Exercise exercise, User user) {
         return participation;
     }
 
+    public StudentParticipation createParticipationWithEmptySubmission(Exercise exercise, User user, SubmissionType submissionType) {
```

Comment:
please add JavaDoc.
You might want to rename this method and append `IfNotExisting`

Reply:
Will add it and perform the renaming.